inc/template-functions.php

```diff
@@ -206,11 +206,26 @@ function twenty_twenty_one_continue_reading_link() {
 }
 
 // Filter the excerpt more link.
-add_filter( 'excerpt_more', 'twenty_twenty_one_continue_reading_link' );
+add_filter( 'excerpt_more', 'twenty_twenty_one_continue_reading_link_excerpt' );
 
-// Filter the content more link.
-add_filter( 'the_content_more_link', 'twenty_twenty_one_continue_reading_link' );
+/**
+ * Create the continue reading link.
+ */
+function twenty_twenty_one_continue_reading_link() {
 
+	if ( ! is_admin() ) {
+		$continue_reading = sprintf(
+			/* translators: %s: Name of current post. */
+			wp_kses( esc_html__( 'Read more %s', 'twentytwentyone' ), array( 'span' => array( 'class' => array() ) ) ),
+			the_title( '<span class="screen-reader-text">' . esc_html__( 'about ', 'twentytwentyone' ), '</span>', false )
+		);
+
+		return '<div><a class="more-link" href="' . esc_url( get_permalink() ) . '">' . $continue_reading . '</a></div>';
```

Comment:
It would be nice to see the link tag generated in another function, so it can be shared between the two `continue_reading` functions, just to ensure any string changes happen in both places.

Reply:
I did not consider that and I agree it would be a better solution.

Reply:
I've added a separate function for the text.